CHANGELOG.md

```diff
@@ -67,6 +67,10 @@ and `worker` details. Even bulk operations such as `insert_all_jobs`, `cancel_al
 
 See the [2.14 upgrade guide](v2-14.html) for step-by-step instructions (all two of them).
 
+## v2.14.2 — Unreleased
+
+- [Oban.Telemetry] - Add `Oban.Telemetry.detach_default_logger/0`
+
```

Comment:
Thanks for adding this item, but I wait to modify the CHANGELOG until I cut the release. Additionally, since this adds a new function it will be in v2.15 next, not a bugfix-only patch release.

```suggestion
```